schemareader/tableFilters.go

```diff
@@ -54,6 +55,47 @@ func applyTableFilters(table Table) Table {
 		virtualIndexColumns := []string{"name", "version"}
 		table.UniqueIndexes[VirtualIndexName] = UniqueIndex{Name: VirtualIndexName, Columns: virtualIndexColumns}
 		table.MainUniqueIndexName = VirtualIndexName
+	case "suseimageinfo":
+		unexportColumns := make(map[string]bool)
+		// Ignore actions relevant only to source server
+		unexportColumns["build_action_id"] = true
+		unexportColumns["inspect_action_id"] = true
+		unexportColumns["build_server_id"] = true
+		table.UnexportColumns = unexportColumns
+		// Unfortunately images have only ID unique and that is not enough for our guessing game.
+		// Create virtual compound index then as close as we can get
+		virtualIndexColumns := []string{"name", "version", "image_type", "image_arch_id", "org_id"}
+		table.UniqueIndexes[VirtualIndexName] = UniqueIndex{Name: VirtualIndexName, Columns: virtualIndexColumns}
+		table.MainUniqueIndexName = VirtualIndexName
+	case "suseimageinfochannel":
+		virtualIndexColumns := []string{"channel_id", "image_info_id"}
+		table.UniqueIndexes[VirtualIndexName] = UniqueIndex{Name: VirtualIndexName, Columns: virtualIndexColumns}
+		table.MainUniqueIndexName = VirtualIndexName
+	case "suseimageprofile":
+		table.PKSequence = "suse_imgprof_prid_seq"
+		// rhnregtoken is completely non-unique standalone, use rhnactivation key instead as reference to the same id
+		references := make([]Reference, 0)
+		for _, r := range table.References {
+			if strings.Compare(r.TableName, "rhnregtoken") == 0 {
+				ref := Reference{}
+				ref.TableName = "rhnactivationkey"
+				ref.ColumnMapping = map[string]string{
+					"token_id": "reg_token_id",
+				}
+				references = append(references, ref)
+			} else {
+				references = append(references, r)
+			}
+		}
+		table.References = references
+	case "susekiwiprofile":
+		virtualIndexColumns := []string{"profile_id"}
+		table.UniqueIndexes[VirtualIndexName] = UniqueIndex{Name: VirtualIndexName, Columns: virtualIndexColumns}
+		table.MainUniqueIndexName = VirtualIndexName
+	case "rhnactivationkey":
```

Comment:
If you don't want to export it, then I think you just need to remove it from the list of export tables on the imagedatadumper.
If this table are needed because of a foreign key, it will be load by default but with Export set to false.